.github/workflows/labeler.yml

```diff
@@ -19,3 +26,44 @@ jobs:
     #   run: |
     #     echo "Running translation scripts"
     #     # Generate .pot -> .po -> .json files
+
+  # This'll look at the PR body and add a label if it contains an issue-closing keyword
+  # it runs on every PR event, including "edited" since people tend to add "closes #123" in the PR body after opening it
+  label-auto-closing:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Log PR Edit
+      run: echo "PR was edited"
+    - name: Auto-label PRs
+      uses: actions/github-script@v7
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
don't think we have access to `secrets` when triggering by `pull_request`. it works on PRs that don't come from forks

Reply:
I had read somewhere that the repo does provide a generated GITHUB_TOKEN with limited authority (tagging, commenting, etc) on forks, so this _should_ work. 

I can add a secrets check to skip the job if the secret is not available, but I think that secret always is.

Reply:
I kinda want to merge it and test the theory. I can add the secret check to skip the action if it doesn't work.

Reply:
I think your right, but default access is readonly: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Hmm... and I see the labeller action above the one I'm adding here has

```
   permissions:
      contents: read
      pull-requests: write
```

... but I'm (a) not sure if _that_ works on forks, or (b) if we can elevate the default permissions like that. 

I'm not sure yet if there's a better approach here, or if there's a way to skip this action for forks if we can't make it work.

Reply:
Curious to know if we are moving forward with this as I am planning to consider it for a long-term proposal to improve the management of the backlog

Reply:
Well, I sure WANT to merge it, but I'm not sure how to proceed. Maybe @mistercrunch knows how to unblock this thing :P

Reply:
Maybe using `pull_request_target`?
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

Reply:
In that case, what happens is it will trigger the action that's defined on the main target repo's branch, and say if you do a checkout it would be the target's ref not the PR one. Does that work here? Actually do you need to checkout?